views/apps/new.html

```diff
@@ -1,241 +1,279 @@
-          />
-          <p
-            class="mt-2 mb-6 text-sm text-gray-500 dark:text-gray-400"
-          >
-            Give your connection a name to easily identify it.
-          </p>
-        </div>
-      {{ else }}
-        <input type="hidden" name="name" value="{{.Name}}" id="name" />
-      {{end}}
+      <input type="hidden" name="ExpiresAt" id="ExpiresAt" value="{{.ExpiresAt}}">
+      <input type="hidden" name="MaxAmount" id="MaxAmount" value="{{.MaxAmount}}">
+      <input type="hidden" name="BudgetRenewal" id="BudgetRenewal" value="{{.BudgetRenewal}}">
+    </div>
+    <button class="block mx-auto p-4 font-medium text-purple-700 text-center my-2 cursor-pointer" onclick="document.getElementById('advanced-settings').classList.toggle('hidden');return false">
+      Advanced Settings
+    </button>
+    {{ if .Pubkey }}
+      <p class="rounded-md text-orange-700 bg-orange-50 p-3 mb-6">
+        ⚠️ Only connect with apps you trust!
+      </p>
+    {{ end }}
+
+    <div class="flex flex-col sm:flex-row sm:justify-center">
+      {{ if not .Pubkey }}
+      <a
+        href="/apps"
+        class="inline-flex bg-white border cursor-pointer dark:bg-surface-02dp dark:border-white/10 dark:hover:bg-surface-16dp duration-150 focus-visible:ring-2 focus-visible:ring-offset-2 focus:outline-none font-medium hover:bg-gray-50 items-center justify-center px-5 py-3 rounded-md shadow text-gray-700 dark:text-neutral-300 transition w-full sm:w-[250px] sm:mr-8 mt-8 sm:mt-0 order-last sm:order-first"
+      >
+        Cancel
+      </a>
+      {{ end }}
+
+      <button
+        type="submit"
+        class="inline-flex w-full sm:w-[250px] bg-purple-700 cursor-pointer dark:text-neutral-200 duration-150 focus-visible:ring-2 focus-visible:ring-offset-2 focus:outline-none font-medium hover:bg-purple-900 items-center justify-center px-5 py-3 rounded-md shadow text-white transition"
+      >
+      {{ if .Pubkey }} Connect Wallet {{ else }} Confirm {{ end }}
+      </button>
+    </div>
+  </form>
+
+  {{if .User.Email}}
+    <p class="mt-8 pt-4 border-t border-gray-300 dark:border-gray-700 text-sm text-gray-500 dark:text-neutral-300 text-center">
+      You're logged in as <span class="font-mono">{{.User.Email}}</span><br>
+    </p>
+  {{end}}
+</div>
 
-      <p class="text-gray-500 dark:text-gray-400 mb-1">
-        <input {{if .Disabled}}tabIndex="-1"{{end}} id="ExpiryCheckbox" type="checkbox" class="w-4 h-4 text-purple-700 bg-gray-50 border border-gray-300 rounded focus:ring-purple-700 dark:focus:ring-purple-600 dark:ring-offset-gray-800 focus:ring-2 dark:bg-surface-00dp dark:border-gray-700">
-        <label for="ExpiryCheckbox" class="ml-1 text-sm font-medium text-gray-900 dark:text-gray-300">Set an expiry date</label>
+<div id="advanced-settings" class="flex fixed inset-0 items-center justify-center hidden" >
```

Comment:
would it be good to move the advanced settings modal to a separate file (template?)

Reply:
we are in simple Go here. doing these things are a bit more complicated here and we don't use that modal anywhere else.

(and I still think the modal is not really good here)

Reply:
Done